script.js

```diff
@@ -85,12 +93,20 @@ function getBeerRec() {
 }
 
 function displayBeers(responseJson, targetedDiv) {
-    for (let i = 0; i < 3; i++) {
-        $(targetedDiv).append(`<div class="beers"><h3>Beer Recomendations based of your recipe....</h3>
-          <h4>${responseJson[i].name}</h4>
-          <img src="${responseJson[i].image_url}" alt="" width="20%">
-          <p>${responseJson[i].description}</p></div>`)
+    // we can't assume there will always be 3 or more results in responseJson
+    // so we can limit our loop to both our upper limit 3 and the length of results
+    let { length = 0 } = responseJson;
```

Comment:
We have a similar situation here where we don't want to make assumptions about the data. The previous code was throwing errors for any beers that didn't have at least 3 results. First, we'll pluck the length off of the responseJson argument defaulting its value to 0.